test/api/simple_test.py

```diff
@@ -0,0 +1,62 @@
+"""Tests for simple use cases of the public api.
+
+These tests should also test the imports, not just the functionality.
+"""
+
+
+def test__api__lint_string():
+    """Basic checking of lint functionality."""
+    import sqlfluff

```

Comment:
Usually, imports are at file (module) scope. Is there a reason these tests are `import`-ing inside the function?

Reply:
I was on the fence about this one, with the initial intent being to also *test the import*, but I guess we test that anyway as part of the test suite. I think I was trying to be too clever - and will revert to standard.